### ISSUE TITLE
Ban support for vulgar sites

### ISSUE THREAD

**Issue by trimechee** [dotnettools/SharpGrabber#73]
Hello, vulgar indecent pornographic sites are contrary to human dignity and pono actresses have spoken of rape and sexual assault suffered, these sites destroy society and the family so it will be good to remove the support for these disgusting sites, thank you !

**Comment by javidsho** [comment]
It contains multiple projects. `SharpGrabber.Deskop` is a classic Windows app. Almost none of them include these websites by default except for `SharpGrabber.Desktop`.
Also, don't worry it's been some time since I removed it from the description and thanks for your advice.

**Comment by trimechee** [comment]
Wonderful ! Amazing ! Thank you very much forthis wsome project : and we should download .NET Core 3.1
Shared libraries of ffmpeg in order that SharpGrabber.Deskop  work? since you had the great kindness and generosity to accept my idea, I think it's better that I delete my first post and delete the complete issue, I don't know if I can do it.....

**Comment by javidsho** [comment]
Yes you should get .net core 3.1 runtime. And if you download it from releases you won't need ffmpeg since it's bundled.
No need to delete this issue it's been closed.

**Comment by trimechee** [comment]
Great, thank you for your precious help !

**Comment by trimechee** [comment]
I have .netframework 4.8 advanced services, integrated and enabled by default in my windows 10, and I check in windows components and .net framework 3.5 is not enabled, is .net framework 4.8 sufficient? and does SharpGrabber use yt-dlp?

**Comment by javidsho** [comment]
No this app doesn't work with .NET Framework at all.
Download .NET Core from here:
https://dotnet.microsoft.com/en-us/download/dotnet/3.1

**Comment by trimechee** [comment]
Wonderful ! Thank you so much !)

**Comment by trimechee** [comment]
Hello, is there a list of sites that SharpGrabber can analyze and extract media content from? or it donwloads only from youtube, vimeo, and instagram? SharpGrabber can fetch and download from streaming websites like these sites without the nedd of usinig video extension and addon like video downloadhelper?

https://wiflix.city/serie-en-streaming/19929-attack-of-the-titans-saison-4.html

https://ourmatch.me/29-04-2022-strasbourg-vs-paris-saint-germain/


http://livetv.sx/enx/video/


https://myfootball.top/index/0-2

**Comment by javidsho** [comment]
It can't just download from any website. Unless you write javascript for that purpose. Check out [BlackWidow](https://github.com/dotnettools/SharpGrabber/tree/master/blackwidow) in that case.  Otherwise, to see what's currently supported check out [wiki](https://github.com/dotnettools/SharpGrabber/wiki/02-Getting-Started).

**Comment by trimechee** [comment]
Interesting ! Thank you very much for your advice :)
